### DISCUSSION TITLE
Rancher Desktop run on startup/login

### DISCUSSION THREAD

**Discussion by madrum** [rancher-sandbox/rancher-desktop#1528] [Q&A] [answered]
I don't see a built-in way to have Rancher Desktop run when I log into Windows.

Does anyone have a recommendation on how to have it run on login?

I've used [NSSM](https://nssm.cc/) to run scripts as Windows Services, so that may be a good option.

Thanks

**Comment by evertonlperes** [comment, marked as answer]
Hi @madrum 
RD does not have auto-run feature yet. Feel free to open a feature request for it [here](https://github.com/rancher-sandbox/rancher-desktop/issues/new?assignees=&labels=kind%2Fenhancement&template=feature_request.yml)

Also, if you're on Win10, here's a quick win tutorial to set up  it for you: [shell:startup](https://support.microsoft.com/en-us/windows/add-an-app-to-run-automatically-at-startup-in-windows-10-150da165-dcd9-7230-517b-cf3c295d89dd)

**Comment by blueboxes** [comment]
This has been raised here https://github.com/rancher-sandbox/rancher-desktop/issues/1254
